docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md

```diff
@@ -15,6 +15,14 @@ Both the ToolJet server and client requires some environment variables to start
 | ------------ | --------------------------------------------------------------- |
 | TOOLJET_HOST | the public URL of ToolJet client ( eg: https://app.tooljet.com )  |
 
+
+#### Tooljet Database (optional)
```

Comment:
Need to also add:

`ENABLE_TOOLJET_DB`
`PGRST_HOST`
`PGRST_JWT_SECRET`

Reply:
added:

ENABLE_TOOLJET_DB
PGRST_HOST
PGRST_JWT_SECRET